packages/state-manager/src/sagas/socket/startConnection/startConnection.saga.ts

```diff
@@ -178,6 +178,7 @@ export function subscribe(socket: Socket) {
       log.error(payload)
       console.error(payload, payload.trace)
       emit(errorsActions.handleError(payload))
+      alert(JSON.stringify(payload))
```

Comment:
I understand that this is only temporary for mobile debugging purposes?
Otherwise I would ask why not use our ErrorModal

Reply:
Yea I was thinking to just deploy something quick so I can get feedback from Kinga on monday. Then we can revert the changes and improve it. The changes will only go to the 2.1.2 branch, not develop. We can also spend more time on it now, but that was my plan at least.